operators/gitlab-operator-kubernetes/0.14.1/metadata/annotations.yaml

```diff
@@ -0,0 +1,15 @@
+annotations:
+  # Core bundle annotations.
+  operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  operators.operatorframework.io.bundle.metadata.v1: metadata/
+  operators.operatorframework.io.bundle.package.v1: gitlab-operator-kubernetes
+  operators.operatorframework.io.bundle.channels.v1: stable,unstable
+  operators.operatorframework.io.bundle.channel.default.v1: stable
+  operators.operatorframework.io.metrics.builder: operator-sdk-v1.23.0
+  operators.operatorframework.io.metrics.mediatype.v1: metrics+v1
+  operators.operatorframework.io.metrics.project_layout: unknown
+
```

Comment:
@gl-distribution-oc please add since your operator is using deprecated api for ocp v4.12 (k8s v1.25)
```
com.redhat.openshift.versions: v4.8-v4.11
```
Also add this annotation to following versions
  - annotationlab:v3.4.0
  - annotationlab:v4.1.2
  - annotationlab:v4.2.0
  - annotationlab:v4.2.1
  - annotationlab:v4.3.0
  - annotationlab:v4.4.0
```
This change can be done in this PR

Reply:
@pursultani  :point_up:

Reply:
It is fixed. In main branch